kong/plugins/datadog/statsd_logger.lua

```diff
@@ -25,8 +25,8 @@ local env_datadog_agent_port = tonumber(os.getenv 'KONG_DATADOG_AGENT_PORT' or "
 
 function statsd_mt:new(conf)
   local sock   = udp()
-  local host = conf.host or env_datadog_agent_host
-  local port = conf.port or env_datadog_agent_port
+  local host = env_datadog_agent_host or conf.host
```

Comment:
Well, I just took another look at this, and I remembered that the AWS lambda plugin:
https://github.com/Kong/kong/blob/f38b38ed53a9e0dbc967a69c6eda1e1d7481fc8d/kong/plugins/aws-lambda/handler.lua#L232

But I think it makes sense here to make the DataDog Agent env a higher priority because the `conf.host` has a default value.

Reply:
^ This would be a breaking change; another approach is to remove the schema default value; both approaches would introduce breaking changes.